### PR TITLE
Minor reformats

### PR DESCRIPTION
**Pull request by filiptronicek**
## Description
Add types, change anonymous functions into arrow ones, and use `const` wherever possible.

## Related Issue(s)

Fixes #

## Release Notes

```release-note
NONE
```

### CODE REVIEW

File: src/gitpodify.ts [hunk 1]
```diff
@@ -70,13 +70,13 @@ const injectedByUserClick = (): boolean => {
...
-                .then(() => isInstalling = false);
-        }
-    });
+    observer = new MutationObserver(() => {
+            if (!injector.checkIsInjected() && !isInstalling) {
+                isInstalling = true;
+                injector.update()
+                    .then(() => isInstalling = false);
+            }
+        });
```
Comment (bigint) [inline src/gitpodify.ts:79]: @filiptronicek I think there are more spaces for indentation 😄 

image

**Review by bigint — CHANGES_REQUESTED** [review]

File: src/utils.ts [hunk 2]
```diff
@@ -5,7 +5,7 @@ export function renderGitpodUrl(gitpodURL: string): string {
 
 export function makeOpenInPopup(a: HTMLAnchorElement): void {
     a.onclick = () => {
-        var w = window.open(a.href, a.target, 'menubar=no,toolbar=no,location=no,dependent');
+        const w = window.open(a.href, a.target, 'menubar=no,toolbar=no,location=no,dependent');
         return !w;
```
Comment (bigint) [inline src/utils.ts:9]: @filiptronicek Can't we make this in single line 🤔

Comment (filiptronicek) [reply src/utils.ts:9]: Working on it 💡

**Review by bigint — APPROVED** [review]
LGTM 🚀

**Review by Psyruss77 — APPROVED** [review]

**Review by gtsiolis — COMMENTED** [review]
Thanks @filiptronicek! UX works as before! 🏁 

@laushinka code changes look trivial and ok to merge but could you take a quick look and confirm we are not missing anything?

**Comment by laushinka** [comment]
Looks good! 👏🏼

/lgtm

**Review by laushinka — APPROVED** [review]

**Comment by laushinka** [comment]
Does this require a manual merge? If yes, feel free to do it!

**Pull request merged**